frontend/src/scenes/notebooks/Notebook/Editor.tsx

```diff
@@ -182,6 +184,49 @@ export function Editor({
 
             onCreate({
                 getJSON: () => editor.getJSON(),
+                getText: () => {
+                    const titleSerializer: TextSerializer = (props): string => props.node.attrs?.title || ''
```

Comment:
Is there no way to do this on the Nodes themselves? Feels like this is very prone to being forgotten about.

Reply:
The typing on the map of serializers will shout if we don't add a custom node here... nothing stopping us asking the node if it wants to override the default though... the implementations were just dumped here for ease while I was figuring out what the what

Reply:
Agree that we should push this to the node if possible because we will want people to be able to customize it for themselves.
I _think_ there is a way to support that via TipTap through node extensions but it's completely new to me so I just spent a little while trying to get it working. Made a separate branch off this one because it was so experimental but something like this commit might work:
https://github.com/PostHog/posthog/pull/17389/commits/7537ef18f673770c6f6e90a64de19b80b5a28c07

Reply:
We also have some text parsing already. It might be good to make sure we coalesce all of these together https://github.com/PostHog/posthog/blob/63d6dbc101b75e29037cddd9a2732cd51fac2c54/frontend/src/scenes/notebooks/Notebook/utils.ts#L89-L96